src/argilla/training/spacy.py

```diff
@@ -267,21 +266,25 @@ def predict(
 
 class ArgillaSpaCyTrainer(_ArgillaSpaCyTrainerBase):
     def __init__(self, freeze_tok2vec: bool = False, **kwargs) -> None:
-        """Initialize the `ArgillaSpaCyTrainer` class.
+        """
+        Initialize the ArgillaSpaCyTrainer class.
 
         Args:
-            freeze_tok2vec: A `bool` indicating whether to freeze the `tok2vec` weights
-                during the training. Defaults to False.
-            **kwargs: The `ArgillaSpaCyTrainerBase` arguments.
+            freeze_tok2vec (bool, optional): A flag indicating whether to freeze the tok2vec weights
```

Comment:
Is the type here needed? I think this may be redundant as we're also defining it in the `__init__` args type-hints? I think we should decide either to keep it for every docstring or remove it from every docstring, but I assume that during the Python reference generation the type-hints are re-used if defined already, so there should be no need

Reply:
e.g.

```suggestion
            freeze_tok2vec: A flag indicating whether to freeze the tok2vec weights
```